packages/types/src/metadata/decorate/storage/util.ts

```diff
@@ -18,8 +20,8 @@ interface ManualDefinition {
   section: string;
 }
 
-function findSiPrimitive (registry: Registry, _prim: string): PortableType | undefined {
-  const prim = _prim.toLowerCase();
+function findSiPrimitive (registry: Registry, type: string): PortableType | undefined {
```

Comment:
nit: `type` is a keyword in Typescript; I'd suggest using a different identifier